test/evdev/test_evdev_init.py

```diff
@@ -0,0 +1,185 @@
+""" Tests for the evdev __init__ module
```

Comment:
I like the fact that you have put effort into tests as well. It keeps pushing the bar. It's something we haven't pushed for in the past but indeed allows to be more confident when others make changes in the future.